ulib/fs/FStar_Int_Cast.fs

```diff
@@ -0,0 +1,180 @@
+#light "off"
+module FStar_Int_Cast
+open Prims
+open FStar_Pervasives
+
+(* !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! *)
+(* THIS FILE WAS EXTRACTED FROM F*  *)
+(* IT HAS TO BE COPIED HERE BECAUSE THIS MODULE IS NOW REALIZED IN
+   OCAML *)
```

Comment:
Wouldn't it be fine to leave both fst and fsti in ulib, and only add OCaml implementation inside ml subdirectory? I'm not 100% sure, but afair F* was smart enough to use the realized module if it was available and otherwise rely on extraction. 

With F#, what is being used is driven with fsproj, but I'm not completely sure how makefiles for OCaml will behave with both fst and ml files present.

Reply:
@mateuszbujalski I didn’t realize I could do that, it's much cleaner that way. Thanks!